sample.env

```diff
@@ -0,0 +1,10 @@
+OPENAPI_KEY="seufhsfewnf-efhewbfbebifoewbfoe"
+DBURL="https://codemark-hbbdskswifeohefoewnwoilcnowncrgu.firebasedatabase.app/"
+FIREBASEAPIKEY="jfdkotaegiiuhKatiyarwljfehlfeis"
+AUTHDOMAIN= "codemark-eukwfgukpandeywbhdwbbdai.firebaseapp.com"
+PROJECTID="codemark-sdfgyiks6iru9vsrgni8dsu8"
```

Comment:
Prefer using FIREBASE as prefix for keys associated with Firebase, also use `_` as separator between words.
eg AUTH_DOMAIN over AUTHDOMAIN

Also just for semantic styling, prefer using spaces between keys and values